.travis.yml

```diff
@@ -1,33 +1,61 @@
 language: php
 
-sudo: false
-
 cache:
   directories:
     - $HOME/.composer/cache/files
 
+branches:
+    only:
+        - master
+
 php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
+  - '5.5'
+  - '5.6'
+  - '7.0'
+  - '7.1'
+  - nightly
 
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env: DEPS=dev
-    - php: 5.5
+    - php: '5.5'
       env: COMPOSER_FLAGS="--prefer-lowest"
+
+    - php: '7.0'
+      env: 'SYMFONY_VERSION=~2.8.0'
     - php: 7.0
-      env: SYMFONY_VERSION=2.7.*
+      env: 'SYMFONY_VERSION=~3.0.0'
     - php: 7.0
-      env: SYMFONY_VERSION=2.8.*
+      env: 'SYMFONY_VERSION=~3.1.0'
+    - php: 7.1
+      env: 'SYMFONY_VERSION=~3.2.0@dev'
+
+    - php: 7.0
+      env: 'ELOQUENT_VERSION=~5.0.0'
+    - php: 7.0
+      env: 'ELOQUENT_VERSION=~5.1.0'
+    - php: 7.0
+      env: 'ELOQUENT_VERSION=~5.2.0'
+    - php: 7.0
+      env: 'ELOQUENT_VERSION=~5.3.0@dev'
+
+  allow_failures:
+    - php: nightly
+    - env: 'ELOQUENT_VERSION=~5.3.0@dev'
```

Comment:
I would be to remove the two lines then. Otherwise, it means you allow builds to fail for an external reason you have no control of (PHP build being unstable)